homeassistant/components/laundrify/icons.json

```diff
@@ -1,8 +1,12 @@
 {
   "entity": {
     "binary_sensor": {
-      "laundrify_power_plug": {
-        "default": "mdi:washing-machine"
+      "wash_cycle": {
+        "default": "mdi:washing-machine",
+        "state": {
+          "on": "mdi:washing-machine",
+          "off": "mdi:washing-machine-off"
+        }
```

Comment:
This change seems unrelated to this PR. Please don't stack multiple changes in a single PR. I suggest to split this out in its own PR.

Reply:
Moved to https://github.com/home-assistant/core/pull/121435 and undone in https://github.com/home-assistant/core/pull/121378/commits/6e0e80a4a44a3a07743e85438716c610ad64652e